.github/ISSUE_TEMPLATE/2_bug-or-performance-issue-report.md

```diff
@@ -0,0 +1,26 @@
+---
+name: Bug or performance issue report
+about: Create a report to help us improve oneDPL
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug or performance issue is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+
+The following information might be useful: 
+ - CMake command: [e.g.cmake -DCMAKE_CXX_COMPILER=icpx -DCMAKE_BUILD_TYPE=release -DONEDPL_PAR_BACKEND=tbb]
+- oneDPL version: [e.g. 2022.6.0 or 224acf7423ded15d34b53429d99248c3d308b685]
+Report oneDPL version and githash. If it is a regression, report githash for the last known good revision.
+ - Compiler: [e.g. Intel(R) oneAPI DPC++/C++ Compiler 2023.1.0 (2023.1.0.20230320)]
```

Comment:
I know we want to keep this as minimal as possible to not overwhelm users, but I think the big thing missing here OS.  It is something we really need to know to have a chance at reproducing the issue.